server/index.js

```diff
@@ -30,11 +30,6 @@ module.exports = function Server(config, onStart) {
     app.log(['serve', 'exit'], 'Goodbye.');
   });
 
-  // Expose ability to implement middleware via API
-  const use = function use() {
-    app.use.apply(app, arguments);
-  };
-
   // Construct and return mockyeah API
-  return Object.assign({ server }, app.routeManager, { use, config, close });
```

Comment:
What is the reasoning to remove the `mockyeah.use()`? This could be a breaking change.

Reply:
I've done some digging... I am not finding good reasoning for `.use()` being included in the mockyeah api. It is undocumented and seems to have creeped into the following commit: https://github.com/ryanricard/mockyeah/commit/fd161aa760a5b0b9099915cd7683cebd2204d006#diff-0861d6d6b50d7d695344bf2d86d6e5e6

Reply:
@ryanricard If we expose `app` via merging Express features into `mockyeah`, we don't need to specially expose `use`. If we go with `mockyeah.app` instead to expose, then we can leave `use` at the top level, unless you want to deprecate it at that level as well.

Reply:
@ryanricard It looks like there was an intent to expose the ability to add custom middleware - see [`test/integration/MiddlewareTest.js`](https://github.com/ryanricard/mockyeah/blob/master/test/integration/MiddlewareTest.js).